docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -363,6 +380,12 @@ others. If an action specifies a skill, this requires a check.
         - *Hide*: Stealth vs. target(s) Detection; impose Lower Hand on attacks against
          you by target(s) who failed
 
+ A character can choose to hold their Major Action to occur on a stated trigger, such
+ as "I will make an Attack, Weapon when an enemy moves within 1 space of me." If the
```

Comment:
I think this should be stated as 'Weapon Attack' but specified as a power as 'Attack, Weapon' so that attacks get sorted together alphabetically.

```suggestion
 as "I will make a Weapon Attack when an enemy moves within 1 space of me." If the
```